README.md

```diff
@@ -101,3 +101,31 @@ query_job = client.query(query)
 for row in query_job.result():
     print(row)
 ```
+
+## Publishing to TestPyPi
+
+Instructions for pushing new versions of `vestapol` to TestPyPi:
+
+1. Update `CHANGELOG.md`. Include Additions, Fixes, and Changes.
+
+2. Update `version` in `pyproject.toml` following [Semantic Versioning](http://semver.org/).
```

Comment:
The [`poetry version`](https://python-poetry.org/docs/master/cli/#version) command can also handle this

Reply:
nice added